src/index.html

```diff
@@ -1,13 +1,330 @@
+              class="products__img"
+            >
+
+            <h3 class="products__text products__text--subtitle">
+              SoundLink Color Bluetooth
+              speaker II
+            </h3>
+
+            <p class="products__text products__text--category">
+              Portable bluetooth
+            </p>
+
+            <p class="products__text products__text--price">
+              $ 129.00
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="category">
+        <h3 class="category__title">
+          Browse Bose
+          <br>
+          products by category
+        </h3>
+        <div class="category__list">
+          <div class="category__item">
+            <div class="category__grid">
+              <div class="category__img category__img--1"></div>
+              <div class="category__img category__img--2"></div>
+            </div>
+
+            <p class="category__text">Headphones & earbuds</p>
+          </div>
+
+          <div class="category__item">
+            <div class="category__grid">
+              <div class="category__img category__img--3"></div>
+              <div class="category__img category__img--4"></div>
+            </div>
+
+            <p class="category__text">Speakers</p>
+          </div>
+
+          <div class="category__item">
+            <div class="category__grid">
+              <div class="category__img category__img--5"></div>
+              <div class="category__img category__img--6"></div>
+            </div>
+
+            <p class="category__text">Audio sunglasses</p>
+          </div>
+        </div>
+
+        <button class="category__button">View all</button>
+      </section>
+
+      <section class="advantage">
+        <h2 class="advantage__title">
+          Why buy direct
+          <br>
+          from Bose
+        </h2>
+
+        <div class="advantage__info">
+          <div class="advantage__points">
+            <p class="advantage__point">
+              Free 2-day shipping and returns
+            </p>
+
+            <p class="advantage__point">
+              90-day risk-free trial
+            </p>
+
+            <p class="advantage__point">
+              World class customer service
+            </p>
+
+            <p class="advantage__point">
+              My Bose account management
+            </p>
+          </div>
+
+          <p class="advantage__detail">
+            A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+          </p>
+        </div>
+      </section>
+
+      <section class="contact">
+        <h2 class="contact__title">
+          Contact us
+        </h2>
+
+        <div class="contact__container">
+          <form class="contact__form">
+            <label for="name" class="contact__label contact__label-name">
```

Comment:
These input labels seem to be redundant

Reply:
I meant to fill them and make invisible. Label serve semantic purpose too. I'll fix the problem.

Reply:
Yes that's right